components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -1530,6 +1530,14 @@ def getMaxPlaneSize(self):
                 int(c.getConfigValue('omero.pixeldata.max_plane_height')))
         return self._maxPlaneSize
 
+    def getRoiLimitSetting(self):
+        try:
+            roi_limit = (self.getConfigService().getConfigValue(
+                         "omero.client.viewer.roi_limit") or 100)
+        except:
+            roi_limit = 100
+        return roi_limit
```

Comment:
Yes, looking at other usage it is cast to an int/long when necessary. Sorry, I forgot to check actually setting the config value as I was using eel.